uint32 cell_x
uint32 cell_y
---
float32 real_x
float32 real_y
